packages/core/contracts/insured-bridge/ovm/Optimism_CrossDomainEnabled.sol

```diff
@@ -76,7 +76,7 @@ contract Optimism_CrossDomainEnabled {
      */
     function sendCrossDomainMessage(
         address _crossDomainTarget,
-        uint32 _gasLimit,
+        uint256 _gasLimit,
```

Comment:
Just making these uint lengths consistent with the Arbitrum messenger's. I'm not worried about gas costs for BridgeAdmin transactions which are not user-facing and will not be frequently called.